src/Eshopworld.Core/Events/IEvent.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Eshopworld.Core.Events
+{
+    /// <summary>
+    /// Standard interface that every event that will be published & consumed from the event streaming bus will expose
+    /// </summary>
+    public interface IEvent
+    {
+        /// <summary>
+        /// Identifier of the event (aggregate's identifier), used as the message key on the bus
+        /// </summary>
+        string Identifier { get; }
+
+        /// <summary>
+        /// User reference that is responsibile for generating the event
+        /// </summary>
+        string CreatedBy { get; }
```

Comment:
Is this a human or a service?

Reply:
The user account reference of the authenticated user that is sending the request to the API which results in the event being emitted

Reply:
why do we care about the user's name? or auditing or something else?

Reply:
Yes, for auditing. If we have both properties (createdon & createdby) we can consume all events on the central auditing solution planned to be presented in ESP using a new micro-frontend